specifyweb/frontend/js_src/lib/components/SchemaConfig/UniquenessRules.tsx

```diff
@@ -0,0 +1,467 @@
+                    ],
+                  })
+                }
+              />
+            ) : null}
+          </>
+        ))}
+        {isExpanded ? (
+          <Button.Icon
+            className={`w-fit ${className.dataEntryAdd}`}
+            icon="plus"
+            title={commonText.add()}
+            onClick={(): void =>
+              handleChanged({
+                ...rule,
+                fields: [
+                  ...rule.fields,
+                  addMissingFields('SpLocaleContainerItem', {}),
+                ],
+              })
+            }
+          />
+        ) : null}
+      </td>
+      <td>
+        <PickList
+          className={isExpanded ? 'w-fit' : ''}
+          disabled={disableRuleModification}
+          groups={{
+            [schemaText.hierarchyScopes()]: relationships
+              .filter((field) =>
+                field.name === databaseFieldName
+                  ? true
+                  : schema.orgHierarchy.includes(
+                      (strictGetTable(container.name)
+                        .getRelationship(field.name)
+                        ?.getReverse()?.table.name ??
+                        '') as typeof schema.orgHierarchy[number]
+                    )
+              )
+              .map((field) => [
+                field.resource_uri,
+                field.strings.name.text,
+              ]) as RA<readonly [string, string]>,
+            [schemaText.advancedScopes()]: relationships
+              .filter((field) =>
+                field.name === databaseFieldName
+                  ? false
+                  : !schema.orgHierarchy.includes(
+                      (strictGetTable(container.name)
+                        .getRelationship(field.name)
+                        ?.getReverse()?.table.name ??
+                        '') as typeof schema.orgHierarchy[number]
+                    )
+              )
+              .map((field) => [
+                field.resource_uri,
+                field.strings.name.text,
+              ]) as RA<readonly [string, string]>,
+          }}
+          value={rule.scope === null ? null : rule.scope.resource_uri}
+          onChange={(value): void => {
+            const newScope =
+              value === null
+                ? null
+                : relationships.at(
+                    relationships.findIndex(
+                      ({ resource_uri }) => resource_uri === value
+                    )
+                  );
+            handleChanged({
+              ...rule,
+              scope: newScope as SerializedResource<SpLocaleContainerItem>,
+            });
+          }}
+        />
+        {isExpanded ? (
+          <Button.Icon
+            className="w-fit"
+            icon="trash"
+            title={commonText.remove()}
+            onClick={handleRemoved}
+          />
+        ) : null}
+        {fetchedDuplicates.totalDuplicates > 0 ? (
```

Comment:
(optional) `{fetchedDuplicates.totalDuplicates > 0 && <Button.Danger...`

Reply:
Fixed in 6f1043eed4a14114cb501aaf721df64bd7d0dcdc